plugins/file-open-save/core/src/main/resources/project.properties

```diff
@@ -0,0 +1,2 @@
+PROJECT_VERSION=8.0-SNAPSHOT

```

Comment:
Given the single usage this has, i'd add a single property with the full client path and fetch it in one go:

CLIENT_PATH=/${project.artifactId}/${project.version}/index.html

Then add this file to src/main/resources-filtered so that it gets dynamically replaced.